### ISSUE TITLE
Ignore sun position higher than threshold

### ISSUE THREAD

**Issue by maramend** [alisdairjsmyth/node-red-contrib-blindcontroller#38]
Hi,

as you have already implemented a functionality to ignore sun position below a threshold (option "Ignore sun position until altitude higher than") it would be great to have the same for the opposite case.

In my scenario I have a roof overhang, which throws a shadow whenever the sun's altitude is at a certain altitude. If the windows are  covered by this shadow this should be taken into account.

**Comment by danlun** [comment]
Yes, This is à god idea if it isn’t solved already with current functionality?

**Comment by HubertusH** [comment]
God idea - some of our windows are "under" the roof and so the sun can not shine into the window when the altitude is high.
